### DISCUSSION TITLE
Was wondering if a date range filter could be added to to create content such as 'Front Page News' or 'Archived News' easier to organize, for example

### DISCUSSION THREAD

**Discussion by Bachynski** [publishpress/publishpress-blocks#1211] [Feature Requests]
I was wondering if a 'Date" filter could be added. It would help with making certain kinds of "Pages", such as a 'Front Page' for anything between a set date and most recently. Or an 'April Archived Page' for any dates between April 1st and 30th, for example. I realize you could filter with Tags or Categories, but then you would have to manually Tag and Categorize. However, I think most people like things simple or automated, and since dating a created post is already automated filtering by date would be easiest to create desired organized content.

**Comment by htmgarcia** [comment]
@Bachynski the "Order by" field takes care of the first suggestion, however I see your point about archives and ranges between 2 dates.
